src/bp-forums/groups.php

```diff
@@ -383,9 +785,19 @@ public function edit_screen_save( $group_id = 0 ) {
 				$forum_id = (int) ( is_array( $forum_ids ) ? $forum_ids[0] : $forum_ids );
 			}
 
-			// Update the group ID and forum ID relationships
-			bbp_update_group_forum_ids( $group_id, (array) $forum_ids );
-			bbp_update_forum_group_ids( $forum_id, (array) $group_id );
+			// Is the forum associated with others group then exclude it.
+			$filter_forum_ids = $this->get_validate_forum_ids( $forum_ids, $group_id );
```

Comment:
change the function name to `validate_forum_ids`

Reply:
Update https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/414560ca0a1366f661b41fee22788390ea057d55